### PR TITLE
Generation of ADF 0.4.0

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

File: azure-mgmt-datafactory/azure/mgmt/datafactory/models/expression.py [hunk 1]
```diff
@@ -37,4 +37,5 @@ class Expression(Model):
     type = "Expression"
 
     def __init__(self, value):
+        super(Expression, self).__init__()
```
Comment (hvermis) [inline azure-mgmt-datafactory/azure/mgmt/datafactory/models/expression.py]: Not sure why these changes are here

File: azure-mgmt-datafactory/azure/mgmt/datafactory/operations/activity_runs_operations.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def internal_paging(next_link=None, raw=False):
             # Construct and send request
             request = self._client.get(url, query_parameters)
             response = self._client.send(
-                request, header_parameters, **operation_config)
+                request, header_parameters, stream=False, **operation_config)
```
Comment (hvermis) [inline azure-mgmt-datafactory/azure/mgmt/datafactory/operations/activity_runs_operations.py]: This one too - not sure why we see this added

File: azure-mgmt-datafactory/azure/mgmt/datafactory/models/expression.py (hunk 1 above)
Comment (lmazuel) [reply azure-mgmt-datafactory/azure/mgmt/datafactory/models/expression.py]: New version of Autorest, this is normal (and was a bug before)

File: azure-mgmt-datafactory/azure/mgmt/datafactory/operations/activity_runs_operations.py (hunk 2 above)
Comment (lmazuel) [reply azure-mgmt-datafactory/azure/mgmt/datafactory/operations/activity_runs_operations.py]: This is an optmisation of latest Autorest. TL;DR; This improve HTTP session pooling.
